zcutil/release-notes.py

```diff
@@ -77,9 +93,17 @@ def generate_release_note(version, filename):
     notes = subprocess.Popen(['git shortlog --no-merges {0}..HEAD'.format(latest_tag)], shell=True, stdin=subprocess.PIPE, stdout=subprocess.PIPE).communicate()[0]
     lines = notes.split('\n')
     lines = [alias_authors_in_release_notes(line) for line in lines]
+    temp_release_note = os.path.join(doc_dir, 'release-notes.md')
     release_note = os.path.join(doc_dir, 'release-notes', 'release-notes-{0}.md'.format(version))
     with open(release_note, 'w') as f:
+        with open(temp_release_note, 'r') as f2:
+            for line in islice(f2, 3, None):
+                f.write(line)
```

Comment:
That's exactly my assumption in this PR: that we will always have at least one notable change. But if you believe that assumption to be invalid, I will update the logic.